ocular/src/chain.rs

```diff
@@ -2,3 +2,58 @@ pub mod client;
 pub mod config;
 pub mod info;
 pub mod registry;
+
+pub type ChainName = tendermint::chain::Id;
```

Comment:
Why not just call this `ChainId` to match the `tendermint` name?

Also note that prefixing a type with the module it's contained in is non-idiomatic:

https://rust-lang.github.io/rfcs/0356-no-module-prefixes.html

I would suggest just re-exporting `tendermint::chain::Id` here:

```suggestion
pub use tendermint::chain::Id;
```

Reply:
My concern is that referring to the reference name as an ID will be conflating the [`chain-id`](https://github.com/cosmos/chain-registry/blob/686044b7c597f9601d4d78a86f3b80bc2c8dfbb9/agoric/chain.json#L7) in the registry with the [chain name](https://github.com/cosmos/chain-registry/blob/686044b7c597f9601d4d78a86f3b80bc2c8dfbb9/agoric/chain.json#L3), thus the rename. That's also why I named it `ChainName`, to match the field in the registry `chains.json`, not to prefix it with the module name. Does that change your opinion?